test/setup.js

```diff
@@ -0,0 +1,3 @@
+// import 'aurelia-polyfills';
+// import {initialize} from 'aurelia-pal-browser';
+// initialize();
```

Comment:
The point of these is to initialize the test browser (for karma). When set up properly, this is useful.